.github/scripts/prepare_data.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# This script takes as input a directory, the name of the archive and a version.
+# It creates an archive in the correct format in the current directory
+# and adds the necessary information to verify its content.
+# The archive will contain the directory name and the provided version.
+# If the option --upload is provided the script will upload the archive to the s3 bucket.
+#
+# ./prepare_data.sh ./bert_models rubert v0 will generate an archive rubert_v0.tgz
+# with one directory rubert_v0 that contains the files that are present in ./bert_models.
+# 
+# If --upload is passed as an option then the file is also uploaded on the S3 bucket.
+
+# directory to prepare for upload
+DIR_PATH=$1
+shift
+NAME=$1
+shift
+VERSION=$1
+shift
+
+while [ $# -gt 0 ]; do
+  opt="$1"
+  shift
+
+  case $opt in
+    --upload)
+    UPLOAD=true
+    ;;
+  esac
+done
+
+DIR_PATH=$(pwd)/$DIR_PATH
+DIR_NAME=$(basename $DIR_PATH)
+ARCHIVE_BASENAME="${NAME}_$VERSION"
+ARCHIVE_NAME="$ARCHIVE_BASENAME.tgz"
+URL="s3://xayn-yellow-bert/$NAME/$ARCHIVE_NAME"
+CHECKSUM_FILE="sha256sums"
+
+CURRENT_DIR=$(pwd)
+cd $DIR_PATH
```

Comment:
:wrench: I'm not sure why this is here when we then switch to a temp dir which is normally out of tree (e.g. in /tmp)

```suggestion
```